packages/core/src/external/saml/xcpd/iti55-envelope.ts

```diff
@@ -8,19 +8,15 @@ import {
   METRIPORT_HOME_COMMUNITY_ID_NO_PREFIX,
   replyTo,
 } from "../../carequality/shared";
-import { OutboundPatientDiscoveryReq } from "@metriport/ihe-gateway-sdk";
-import { wrapIdInUrnUuid } from "../utils";
+import { OutboundPatientDiscoveryReq, XCPDGateway } from "@metriport/ihe-gateway-sdk";
+import { wrapIdInUrnUuid, timestampToSoapBody } from "../utils";
 
 const DATE_DASHES_REGEX = /-/g;
-
+const expiresIn = 5;
```

Comment:
What's the unit here? We've been using dayjs' duration to avoid having to think/wonder about it.

Reply:
Marked as resolved but it still doesn't have the unit. 😄